python/tests/test_bradley_mapping.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+import sys
+import os
+import json
+
+
+# Add the config directory to the sys.path
+sys.path.append(
```

Comment:
I agree this is messy, but don't offhand know the solution.  See if making this code more package-like with `__init__.py` files helps, but for now this is secondary to the suggested code restructuring elsewhere.